boards/xtensa/esp32/doc/esp32.rst

```diff
@@ -165,6 +168,30 @@ pinout.  Power to the ESP32 board should be provided by a USB cable.
 | IO15       | TDO       |
 +------------+-----------+
 
+Once the device is connected, you should be able to connect with (for
```

Comment:
how about:

     Once the device is connected, you should be able to connect with the
     following commands.  Note that for a DevKitC board,
     replace ``board/esp-wroom-32.cfg`` in the command line below with
     ``board/exp32-wrover.cfg`` (for  WROVER):